xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/export_modal.vm

```diff
@@ -4,70 +4,82 @@
  * Add export modal
  *#
 #macro(exportModal)
-  <div class="modal fade text-left" id="exportModal" tabindex="-1" role="dialog" aria-labelledby="exportModalLabel" aria-hidden="true">
-    <div class="modal-dialog">
-      <div class="modal-content">
-        <div class="modal-header">
-          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-          <h4 class="modal-title" id="exportModalLabel">$services.localization.render('core.menu.export')</h4>
-        </div>
-        <div class="modal-body">
-          <div class="panel-group" id="exportModelAccordion" role="tablist" aria-multiselectable="true">
-            #set($exportExtensions = $services.uix.getExtensions('org.xwiki.plaftorm.menu.export.buttons'))
+  <div class="modal fade text-left" id="exportModal" tabindex="-1" role="dialog" aria-labelledby="exportModalLabel"
+     aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title" id="exportModalLabel">$services.localization.render('core.menu.export')</h4>
+      </div>
+      <div class="modal-body">
+        <div class="panel-group" id="exportModelAccordion" role="tablist" aria-multiselectable="true">
+          #set($exportExtensions = $services.uix.getExtensions('org.xwiki.plaftorm.menu.export.buttons'))
```

Comment:
The HTML was already nicely indented. I don't understand why you removed the indentation.

Reply:
Regarding all indentation problem: once I fixed my IDE I used the "format code" which completely mess up the diff. My bad.
I'll use the first version of the file and apply back my changes to fix that.